port/common/omrsysinfo.c

```diff
@@ -936,3 +936,49 @@ omrsysinfo_cgroup_is_memlimit_set(struct OMRPortLibrary *portLibrary)
 {
 	return FALSE;
 }
+
+/**
+ * Gets the file pointer by opening the file passed as a param fileName
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ *
+ * @param[in] subsystemFlag for checking the index of the subsystem
+ *
+ * @param[in] fileName pointer to cgroup subsystem filename
+ *
+ * @param[in] file pointer to the file pointer
+ *
+ * @return 0 on success and OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM on failure
+ */
+int32_t
+omrsysinfo_cgroup_get_handle_subsystem_file(struct OMRPortLibrary *portLibrary,  uint64_t subsystemFlag, const char *fileName, FILE **file)
```

Comment:
I don't think use of `FILE` will be acceptable (but I'm not a committer); I expect you'll want to use `intptr_t` instead (the type returned by `omrfile_open`).

Reply:
Yes omrfile_open returns an intptr_t so that is what should be used as the parameter for this function.  ALL port library APIs should be visible on all platforms so types like FILE require a generic type to be used.

Reply:
Hi @charliegracie @keithc-ca I have made changes accordingly. Thanks you for the review comments